esbuildConfig.js

```diff
@@ -0,0 +1,128 @@
+const { sassPlugin, postcssModules } = require('esbuild-sass-plugin');
+const autoprefixer = require('autoprefixer');
+const postcss = require('postcss');
+const yargsParser = require('yargs-parser');
+const { writeFileSync } = require('fs');
+const { resolve, join } = require('path');
+const alias = require('esbuild-plugin-alias');
+
+// from https://github.com/bvaughn/react-virtualized/issues/1212#issuecomment-847759202 workaround for https://github.com/bvaughn/react-virtualized/issues/1632 until it is released.
+const resolveFixup = {
+    name: 'resolve-fixup',
+    setup(build) {
+        build.onResolve({ filter: /react-virtualized/ }, async (args) => {
+            return {
+                path: resolve(join(__dirname, './packages/ui-components/node_modules/react-virtualized/dist/umd/react-virtualized.js'))
+            };
+        });
+    }
+};
+
+const commonConfig = {
+    write: true,
+    bundle: true,
+    metafile: true,
+    sourcemap: true, // .vscodeignore ignores .map files when bundling!!
+    minify: true,
+    logLevel: 'warning',
+    loader: {
+        '.jpg': 'file',
+        '.gif': 'file',
+        '.mp4': 'file',
+        '.graphql': 'text',
+        '.png': 'file',
+        '.svg': 'file'
+    },
+
+    external: [
```

Comment:
Do you really have any references to these modules?

Reply:
removed. it was copied from internal repo and not cleanup